debezium-core/src/main/java/io/debezium/transforms/UnwrapFromEnvelope.java

```diff
@@ -58,24 +59,61 @@
             .withDescription("Drop delete records converted to tombstones records if a processing connector "
                     + "cannot process them or a compaction is undesirable.");
 
+    private static final Field HANDLE_DELETES = Field.create("delete.handling.mode")
+            .withDisplayName("Handle delete records")
+            .withType(ConfigDef.Type.STRING)
```

Comment:
Please use enum insted of string, check `io.debezium.connector.mongodb.transforms.UnwrapFromMongoDbEnvelope.ARRAY_ENCODING` for an example.

Reply:
OK.